Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [evcc/evcc](https://hosted.weblate.org/projects/evcc/evcc/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/evcc/-/evcc/horizontal-auto.svg)

Commit:
Translated using Weblate (German)

Currently translated at 77.1% (132 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

andig commented:
@naltatis hast Du dir Strings überarbeitet?

RTTTC committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/

RTTTC committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/

naltatis commented:
No, nicht meine Änderungen. Autor ist @kammerl.olaf
https://hosted.weblate.org/changes/?project=evcc&component=evcc&lang=de

naltatis commented on i18n/de.toml:
```diff
@@ -29,17 +29,17 @@ mi = "Meilen"
-modalUpdateStarted = "Nach der Aktualisierung wird evcc neu gestartet."
-modalInstalledVersion = "Aktuell installierte Version"
-modalNoReleaseNotes = "Keine Releasenotes verfügbar. Mehr Informationen zur neuen Version findest du hier:"
+availableShort = "Installieren"
```
Das ist die Kurzversion von "Neue Version verfügbar". Hier kann Update nicht mit Installieren übersetzt werden.

premultiply replied:
Könntest du die deutsche und englische Version bei Weblate nochmal kritisch prüfen?
Mit dem großen EN-Update durch den PR (ohne Weblate) sind meiner Ansicht nach viele sprachliche Vereinfachungen reingekommen die ggf. nicht immer so treffend sind und jetzt die Basis für weitere Sprachanpassungen waren.

andig commented:
Ich bin von den Änderungen nicht überzeugt, Review müsste aber bei Weblate passieren. Wie wollen wir vorgehen?

RTTTC committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

RTTTC committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/

premultiply commented:
Änderungen bitte einfach direkt bei Weblate (rückgängig) machen und ggf. den Kontext ergänzen.

luebbe committed:
Translated using Weblate (German)

Currently translated at 80.7% (138 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

andig commented:
Denke auch- Weblate als master. Ich muss mich vom Review aber abmelden- dafür fehlt aktuell die Kapa.

premultiply commented:
Passt scho. Werde ein Auge drauf halten.

roto11r committed:
Translated using Weblate (French)

Currently translated at 61.4% (105 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

premultiply committed:
Translated using Weblate (German)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

premultiply committed:
Translated using Weblate (Luxembourgish)

Currently translated at 85.3% (146 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lb/

weblate committed:
Merge remote-tracking branch 'origin/master'

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

menkej committed:
Translated using Weblate (Spanish)

Currently translated at 63.7% (109 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

menkej committed:
Translated using Weblate (Spanish)

Currently translated at 66.6% (114 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

menkej commented:
Klinke mich hier mal mit ein. Wenn ihr über Deutsch/Englisch geschaut habt, kann ich auch nochmal Spanisch föhnen. Hatte eben schon angefangen, aber dann erst gesehen, dass ihr mit den (neuen) englischen Texten teilweise nicht glücklich seid...

premultiply commented:
Ist halt jetzt ein "Work-in-Progress".
Da müssen sich alle irgendwie erstmal einfinden und einruckeln. 🙂
Jedenfalls ist es schon fantastisch welche neuen Sprachen jetzt dadurch in kurzer Zeit dazu kamen und auch gepflegt und gemonitort werden können.

Wird schon irgendwie... 🙂

menkej commented:
Absolut! Ich kannte weblate bisher auch nicht. Ist schon eine echt feine Sache!

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

nelli81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

int5749 committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 57.8% (99 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/

int5749 committed:
Translated using Weblate (Italian)

Currently translated at 70.1% (120 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/

weblate committed:
Merge remote-tracking branch 'origin/master'

int5749 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/

int5749 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/

int5749 committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/

int5749 committed:
Translated using Weblate (Dutch)

Currently translated at 76.0% (130 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/

int5749 committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

int5749 committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

int5749 committed:
Translated using Weblate (German)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

int5749 committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/

int5749 committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/

int5749 committed:
Translated using Weblate (Dutch)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/

int5749 commented:
Hallo zusammen,

ich habe heute mal etwas Zeit spendiert und bei Weblate einige Sprachen an die neue Struktur der englischen Basis angeasst.
Im Detail: Französisch, Italienisch, Niederländisch, Norwegisch, Spanisch
Evtl. prüft dies noch jemand mit einem zweiten Auge?

In der englischen Basis hatte ich einen Fehler gefunden
`
"You get stickers and digital confetti
`
Dort fehlte das abschließende `"` Da ich nicht wusste, ob der `;)` auch noch dort hin sollte, habe ich diesen erst einmal weggelasen.
Im Anschluß habe ich dies dann auch in allen Sprachen korrigiert.

Gibt es feste Zuordnungen wegen Sprach-Updates? Hoffe mein Aktionismus passte.

int5749 commented:
> Klinke mich hier mal mit ein. Wenn ihr über Deutsch/Englisch geschaut habt, kann ich auch nochmal Spanisch föhnen. Hatte eben schon angefangen, aber dann erst gesehen, dass ihr mit den (neuen) englischen Texten teilweise nicht glücklich seid...

Oups, gerade erst gelesen. Habe Spanisch eben mal nach bestem Wissen und Gewissen bereinigt. Aber ein weiterer Review ist immer hilfreich ;-)

zigad committed:
Added translation using Weblate (Slovenian)

zigad committed:
Translated using Weblate (Slovenian)

Currently translated at 3.5% (6 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sl/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

premultiply committed:
Translated using Weblate (German)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

premultiply committed:
Translated using Weblate (Luxembourgish)

Currently translated at 85.3% (146 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lb/

int5749 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/

int5749 committed:
Added translation using Weblate (Croatian)

int5749 committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/hr/

int5749 commented:
@premultiply 
Ich habe gerade eine Übersetzung für kroatisch angelegt. Dabei ist mir aufgefallen, dass das Template auch noch falsch ist:
`"You get stickers and digital confetti`  ` ;)"`
Habe noch keine Möglichkeit gefunden dies anzupassen, kannst Du dies durchführen?

premultiply committed:
Translated using Weblate (Luxembourgish)

Currently translated at 85.3% (146 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lb/

premultiply committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/

premultiply committed:
Translated using Weblate (German)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

premultiply committed:
Translated using Weblate (French)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

premultiply committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/

premultiply committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/hr/

premultiply committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/

premultiply committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/

premultiply committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/

premultiply committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/

premultiply committed:
Translated using Weblate (Portuguese)

Currently translated at 57.8% (99 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pt/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

premultiply committed:
Translated using Weblate (Lithuanian)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/

premultiply committed:
Translated using Weblate (Portuguese)

Currently translated at 57.8% (99 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pt/

premultiply committed:
Translated using Weblate (Spanish)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/

premultiply committed:
Translated using Weblate (Italian)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/

premultiply committed:
Translated using Weblate (Polish)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

premultiply committed:
Translated using Weblate (French)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/

premultiply committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

Commit:
Added translation using Weblate (Ukrainian)

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 4.6% (8 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/uk/

premultiply committed:
Translated using Weblate (Ukrainian)

Currently translated at 23.9% (41 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 23.9% (41 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/uk/

premultiply committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/

premultiply committed:
Translated using Weblate (English)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/en/

premultiply committed:
Translated using Weblate (German)

Currently translated at 99.4% (170 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (171 of 171 strings)

Translation: evcc/evcc
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/uk/

andig commented on i18n/de.toml:
```diff
@@ -61,28 +61,28 @@ savingsTotalEnergy = "{total} kWh geladen"
 titleSponsor = "Du bist Unterstützer"
 titleNoSponsor = "Unterstütze uns"
-thanks = "Danke für dein Sponsoring, {sponsor}! Das hilft uns bei der Weiterentwicklung."
+thanks = "Vielen Dank, {Sponsor}! Dein Beitrag hilft, evcc weiterzuentwickeln."
```
Seit neuestem ja

    weiter zu entwickeln

premultiply replied:
Dann bitte in Weblate anpassen.
